src/pages/home/report/ReportActionCompose/ComposerWithSuggestions/ComposerWithSuggestions.js

```diff
@@ -105,6 +106,7 @@ function ComposerWithSuggestions({
     const styles = useThemeStyles();
     const {preferredLocale} = useLocalize();
     const isFocused = useIsFocused();
+    const [firstChange, setFirstChange] = useState(true);
```

Comment:
Is there a particular reason you are using `useState` instead of `useRef`? Since `firstChange` isn't used in rendering, I think using a ref would be more appropriate.

Reply:
`firstChange` isn't a very good variable name. What about something like `hasCommentBeenUpdatedOnce`? Though, I'm not really following the code about why this variable is necessary or what the intention of it is.

Reply:
Good point, I think `composerIsEmpty` could make more sense, please let me know what you think.